src/App.tsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type AppProps = {};
+
+type AppState = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<AppProps, AppState> {
```

Comment:
empty object does not deserve the type 😉 
```suggestion
type AppState = {
  pressedKey: string;
};

export class App extends React.Component<{}, AppState> {
```